README.md

```diff
@@ -1504,7 +1504,21 @@ var event = await query.executeQuery();
 console.log(`>> Event: ${JSON.stringify(event)}`);
 ```
 
-
+# File Upload
+This API is used to upload a file and return the metadata that needs to be saved in delivery. Currently, this API is supported only in Browsers. 
```

Comment:
* return => returns (typo)
* upload is a generic mechanism. It _can_ be used in deliveries, but is not limited to deliveries. Your comment seems to imply that it does.
* Browsers => browsers (no need for upper case)